src/js/index.js

```diff
@@ -181,6 +181,11 @@ export const hpe = deepFreeze({
       background: 'active-background',
       color: 'active-text',
     },
+    dateInput: {
```

Comment:
This looks like it is in global. Does it need to be move after checkBox and before formField?

Reply:
oh yes just moved!